docs/source/user-guide/configs.md

```diff
@@ -35,32 +35,33 @@ Values are parsed according to the [same rules used in casts from Utf8](https://
-| datafusion.optimizer.prefer_hash_join                     | Boolean | true    | When set to true, the physical plan optimizer will prefer HashJoin over SortMergeJoin. HashJoin can work more efficientlythan SortMergeJoin but consumes more memory. Defaults to true                                                                                                                                                        |
-| datafusion.optimizer.repartition_aggregations             | Boolean | true    | Should DataFusion repartition data using the aggregate keys to execute aggregates in parallel using the provided `target_partitions` level                                                                                                                                                                                                    |
-| datafusion.optimizer.repartition_joins                    | Boolean | true    | Should DataFusion repartition data using the join keys to execute joins in parallel using the provided `target_partitions` level                                                                                                                                                                                                              |
-| datafusion.optimizer.repartition_windows                  | Boolean | true    | Should DataFusion repartition data using the partitions keys to execute window functions in parallel using the provided `target_partitions` level                                                                                                                                                                                             |
-| datafusion.optimizer.skip_failed_rules                    | Boolean | true    | When set to true, the logical plan optimizer will produce warning messages if any optimization rules produce errors and then proceed to the next rule. When set to false, any rules that produce errors will cause the query to fail.                                                                                                         |
-| datafusion.optimizer.top_down_join_key_reordering         | Boolean | true    | When set to true, the physical plan optimizer will run a top down process to reorder the join keys. Defaults to true                                                                                                                                                                                                                          |
+| key                                                       | default | description                                                                                                                                                                                                                                                                                                |
```

Comment:
The major difference here seems to be the type column is no longer present. Was that intended?

Reply:
Yeah, the fields are now Rust typed not datatype. I could include this, but it seemed unnecessary?

Reply:
It is probably clear from the names if something is boolean or numeric. I don't feel strongly